index.html

```diff
@@ -0,0 +1,234 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Africa Youth summit 2022</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap"
+      rel="stylesheet"/>
+    <link rel="stylesheet" href="styless.css" />
+    <meta
+      name="apple-mobile-web-app-status-bar-style"
+      content="black-translucent"/>
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet">
+    <link rel="stylesheet" href="style.css"/>
+    <link rel="stylesheet" href="about.html"/>
+ 
+  </head>
+  <body>
+    <header class="header">
+      <div class="top-bar">
+        <ul class="top-bar__menu">
+          <li class="top-bar__list">
+            <a href="#" class="top-bar__links">
+              <i class="fab fa-facebook-f"></i>
+            </a>
+          </li>
+          <li class="top-bar__list">
+            <a href="#" class="top-bar__links">
+              <i class="fab fa-twitter"></i>
+            </a>
+          </li>
+          <li class="top-bar__list">
+            <a href="#" class="top-bar__links"> English </a>
+          </li>
+          <li class="top-bar__list">
+            <a href="#" class="top-bar__links"> My Page </a>
+          </li>
+          <li class="top-bar__list">
+            <a href="#" class="top-bar__links"> Logout </a>
+          </li>
+        </ul>
+      </div>
+      <nav class="nav">
+        <div class="nav__container">
+          <a href="about.html" class="logo">
+            <img
+              src="./images/africalogo.jpg"
+              alt="Africa logo"
+              class="logo-img"
+            />
+          </a>
+
+          <div class="nav__mobile">
+            <input type="checkbox" id="check" class="nav__mobile--check" />
+            <label for="check" class="nav__mobile--label">
+              <span class="nav__mobile--icon">&nbsp;</span>
+            </label>
+
+            <div class="nav__mobile--sidebar">
+              <div class="nav__mobile--header">
+                <span>Quick Links</span>
+              </div>
+              <ul class="nav__mobile--menu">
+                <li class="nav__mobile--list">
+                  <a href="about.html#aboutto" class="nav__mobile--links">About</a>
+                </li>
+                <li class="nav__mobile--list">
+                  <a href="index.html#program" class="nav__mobile--links">Program</a>
+                </li>
```

Comment:
- [ ] [Optional] You can use different links in each one of the pages, for example, this one could be exclusive of `index.html` and be excluded of `about.html`